packages/protocol-kit/tests/e2e/passkey.test.ts

```diff
@@ -127,296 +127,347 @@ describe('Passkey', () => {
     }
   })
 
-  describe('isOwner', async () => {
-    it('should fail if the Safe is not deployed', async () => {
+  describeif(safeVersionDeployed < '1.3.0')('init', async () => {
+    it('should fail for Safe versions lower then 1.3.0', async () => {
       const {
         predictedSafe,
         contractNetworks,
         provider,
         passkeys: [passkey1]
       } = await setupTests()
-      const safeSdk = await Safe.init({
-        provider,
-        predictedSafe,
-        contractNetworks,
-        signer: passkey1
-      })
-      chai.expect(safeSdk.isOwner(passkey1)).to.be.rejectedWith('Safe is not deployed')
-    })
 
-    it('should return true if passkey signer is an owner of the connected Safe', async () => {
-      const {
-        contractNetworks,
-        provider,
-        passkeys: [passkey1],
-        passkeySigners: [passkeySigner1]
-      } = await setupTests()
-      const passkeySigner1Address = await passkeySigner1.getAddress()
-      const safe = await getSafeWithOwners([passkeySigner1Address])
-
-      const safeSdk = await Safe.init({
-        provider,
-        safeAddress: await safe.getAddress(),
-        contractNetworks,
-        signer: passkey1
-      })
-
-      const isOwner = await safeSdk.isOwner(passkey1)
-      chai.expect(isOwner).to.be.true
+      chai
+        .expect(
+          Safe.init({
+            provider,
+            predictedSafe,
+            contractNetworks,
+            signer: passkey1
+          })
+        )
+        .to.be.rejectedWith(
+          'Current version of the Safe does not support the Passkey signer functionality'
+        )
     })
+  })
 
-    it('should return false if an account is not an owner of the connected Safe', async () => {
+  describe('isOwner', async () => {
+    itif(safeVersionDeployed < '1.3.0')(
+      'should fail for a passkey argument of Safe <v1.3.0',
```

Comment:
```suggestion
      'should fail for Safe versions lower than 1.3.0',
```

Reply:
I was thinking a bit about this test. The error is thrown by the `init` method so the truth is that we do twice the same test, as we check this in the previous one.

Reply:
Yes. The reason for that is that I came up with the test without looking too much at the implementation at first. I just thought:
- A protocol-kit instance cannot be valid if there is a passkey signer on a version lower than 1.3.0
- A valid protocol-kit cannot handle a `isOwner` query with passkey as an argument.

Reply:
Ok, after your latest changes then updating this test description would be enough